manager_contacts/src/ContactsManager.java

```diff
@@ -0,0 +1,22 @@
+public class ContactsManager {
+    Contact [] myFriends;
+    int friendsCount;
+
+    ContactsManager(){
+        this.friendsCount = 0;
+        this.myFriends = new Contact[500];
+    }
+
+    void addContact(Contact contact){
+        myFriends[friendsCount] = contact;
```

Comment:
@valetodo You don't have a validation in case you are adding the contact 501, it will cause an exception